recipe/meta.yaml

```diff
@@ -35,21 +37,21 @@ requirements:
     - cuda-version {{ cuda_compiler_version }}
   run:
     - {{ pin_compatible('custatevec', max_pin='x') }}
-    - pennylane >=0.32
-    - python
     - {{ pin_compatible("cuda-version", min_pin="x", max_pin="x") }}
+    - cudatoolkit                                       # [(cuda_compiler_version or "").startswith("11")]
+    - {{ pin_compatible('cuda-cudart', max_pin='x') }}  # [(cuda_compiler_version or "").startswith("12")]
+    - {{ pin_compatible('libcublas', max_pin='x') }}    # [(cuda_compiler_version or "").startswith("12")]
+    - {{ pin_compatible('libcusparse', max_pin='x') }}  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
As these should already be added by the `*-dev` packages above, don't think we need these here

```suggestion
```

Reply:
yes for cuda 12 these are not needed (https://github.com/conda-forge/pennylane-lightning-gpu-feedstock/pull/14#discussion_r1422737669), but we still have the multiple package found warning without this